libs/community/langchain_community/vectorstores/azure_cosmos_db.py

```diff
@@ -121,7 +121,7 @@ def from_connection_string(
         connection_string: str,
         namespace: str,
         embedding: Embeddings,
-        application_name: str = "LANGCHAIN_PYTHON",
+        application_name: str = "LangChain-CDBMongoVCore-VectorStore-Python",
```

Comment:
```suggestion
        application_name: str = "LANGCHAIN_PYTHON",
```
these default changes are unfortunately likely breaking - will bring them back to the old one! When moving to the partner package, you're welcome to update